wsdl/ver10/schema/onvif.xsd

```diff
@@ -1462,6 +1462,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 					<xs:documentation>True if the metadata stream shall contain the PTZ position</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="FieldOfView" type="xs:boolean">
```

Comment:
There is no extension point here (xs:any)

Reply:
So you can't add an element. There is an attribute extension point so it would work to add it as an attribute.

Reply:
Also an extension always has to be optional (i.e. mnOccurs="0")

Reply:
Ack. Will fix.